app/assets/javascripts/password-strength-indicator.js

```diff
@@ -17,7 +16,7 @@
             passwordConfirmation = $(options["password_confirmation_field"]).val();
         instance.updateIndicator(password, passwordConfirmation, options);
       };
-      $(password_field).debounce("keyup", update, 50);
+      $(password_field).on("blur", update);
```

Comment:
I think we might want to keep this - I had trouble resetting my password. I first entered a weak password and I tried to update it, but the error message persisted no matter my changes. Eventually I got something working, but it seemed quite random why it accepted the password. Only after seeing this change I realised that the validation happens on blur.